components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md

```diff
@@ -112,7 +112,6 @@ sudo sed -i 's/SELINUX=enforcing/SELINUX=permissive/g' /etc/selinux/config
    - Give `ssh_user` which has access to all the machines. Example: `ubuntu`
    - Give `ssh_port` in case your AMI is running on custom ssh port, default will be 22.
    - Give the `ssh_key_file` path; this key should have access to all the Machines or VMs.
-   - `sudo_password` is only meant to switch to sudo user. If you have configured a password for the sudo user, please provide it here.
```

Comment:
You have removed this here, but did not add link to Sudo password page anywhere, how will they know what to do if they have sudo password needed.

Reply:
done